### ISSUE TITLE
Installation

### ISSUE THREAD

**Issue by gustiii** [Jaykul/Jupyter-PowerShell#5]
Hii,
This is very nice! 
I have some Problems with the installing, I get a dead kernel...
Did you know the dependencies

**Comment by Jaykul** [comment]
I think I packaged the .net core version wrong: if you have a "publish" folder in your "PowerShell-Core" folder, you need to move _all_ of it's contents up one level. I was going to publish a fix, but time has gotten away from me, so it's going to have to be tomorrow.

**Comment by gustiii** [comment]
When I execute the install I get this msg:
```

WARNUNG:
            The currently installed 'dotnet' (.NET Command Line Tools) is not the expected version.

            Installed version: 2.0.0-preview2-006497
            Expected version: 2.0.0-preview1-005977

            If your version is older, you probably cannot run the 'PowerShell (Core)' kernel.

            1. Remove the installed version from:
                - on windows '$env:LOCALAPPDATA\Microsoft\dotnet'
                - on osx and linux '$env:HOME/.dotnet'
            2. Re-run this script with -InstallDotnet

```

**Comment by Jaykul** [comment]
Oh, shoot. That error message is out of date anyway because I removed the "InstallDotNet" feature.
 
1. If you're on Windows, you can safely ignore that, just use the "PowerShell (Full)" kernel.
2. If you need to use "PowerShell (Core)" then you need to get the latest [preview of DotNet Core 2](https://www.microsoft.com/net/core/preview) and install it.

**Comment by Jaykul** [comment]
I think I've fixed the release package in [v1.0.0-beta-4](https://github.com/Jaykul/Jupyter-PowerShell/releases/tag/v1.0.0-beta-4) so the dlls are all in the right folder(s) for Core. 

### You still have to have the latest preview of dotnet and PowerShell Core if you want to use those.

**Comment by vors** [comment]
Getting the same error message:
```
[W 23:42:40.664 NotebookApp] Timeout waiting for kernel_info reply from 36693c62-e856-44dc-b599-ae47f2d39c93
[I 23:42:42.622 NotebookApp] KernelRestarter: restarting kernel (4/5), new random ports
WARNING:root:kernel 36693c62-e856-44dc-b599-ae47f2d39c93 restarted
It was not possible to find any compatible framework version
The specified framework 'Microsoft.NETCore.App', version '2.0.0-preview3-25426-01' was not found.
  - Check application dependencies and target a framework version installed at:
      /usr/local/share/dotnet/
  - The .NET framework can be installed from:
      https://aka.ms/dotnet-download-runtime
  - The .NET framework and SDK can be installed from:
      http://go.microsoft.com/fwlink/?LinkID=798306&clcid=0x409
  - The following versions are installed:
      2.0.0 at [/usr/local/share/dotnet/shared]
      2.1.0-preview1-26001-02 at [/usr/local/share/dotnet/shared]
      2.0.3 at [/usr/local/share/dotnet/shared]
```

I was trying to install the required runtime and it looks like it was deleted (fun with dotnet versions, don't you love it?)

```
PS /Users/vors/dev/Jupyter-PowerShell> /Users/vors/Downloads/dotnet-install.sh -v 2.0.0-preview3-25426-01                   
dotnet-install: Downloading link: https://dotnetcli.azureedge.net/dotnet/Sdk/2.0.0-preview3-25426-01/dotnet-sdk-2.0.0-preview3-25426-01-osx-x64.tar.gz
curl: (22) The requested URL returned error: 404 
dotnet-install: Cannot download: https://dotnetcli.azureedge.net/dotnet/Sdk/2.0.0-preview3-25426-01/dotnet-sdk-2.0.0-preview3-25426-01-osx-x64.tar.gz
dotnet-install: Downloading legacy link: https://dotnetcli.azureedge.net/dotnet/Sdk/2.0.0-preview3-25426-01/dotnet-dev-osx-x64.2.0.0-preview3-25426-01.tar.gz
curl: (22) The requested URL returned error: 404
```

Can we just update references to 2.0.0 or some newer version?

**Comment by vors** [comment]
FWIW after updating to 2.0.3 and rebuilding locally

```

Unhandled Exception: System.TypeInitializationException: The type initializer for 'System.Management.Automation.Utils' threw an exception. ---> System.TypeInitializationException: The type initializer for 'Microsoft.Win32.Registry' threw an exception. ---> System.PlatformNotSupportedException: Registry is not supported on this platform.
   at Microsoft.Win32.RegistryKey.OpenBaseKeyCore(RegistryHive hKey, RegistryView view)
   at Microsoft.Win32.Registry..cctor()
   --- End of inner exception stack trace ---
   at System.Management.Automation.Platform.get_IsNanoServer()
   at System.Management.Automation.Platform.get_IsInbox()
   at System.Management.Automation.Utils..cctor()
   --- End of inner exception stack trace ---
   at System.Management.Automation.Utils.get_DefaultPowerShellAppBase()
   at System.Management.Automation.PSSnapInReader.ReadRegistryInfo(Version& assemblyVersion, String& publicKeyToken, String& culture, String& architecture, String& applicationBase, Version& psVersion)
   at System.Management.Automation.PSSnapInReader.ReadCoreEngineSnapIn()
   at System.Management.Automation.Runspaces.InitialSessionState.ImportCorePSSnapIn()
   at System.Management.Automation.Runspaces.InitialSessionState.CreateDefault2()
   at Jupyter.PowerShell.PowerShellEngine..ctor(PowerShellOptions options, ILogger logger) in /Users/vors/dev/Jupyter-PowerShell/PowerShell-Kernel/PowerShellEngine.cs:line 29
   at Jupyter.PowerShell.Kernel.Main(String[] args) in /Users/vors/dev/Jupyter-PowerShell/PowerShell-Kernel/Kernel.cs:line 70

```

**Comment by vors** [comment]
Interestingly, my old hacky kernel https://github.com/vors/jupyter-powershell works after replacing `powershell` by `pwsh` :)

**Comment by vors** [comment]
UPD: dig a little bit more into it and realized that windows-only nuget package is a problem https://github.com/PowerShell/PowerShell/issues/3417

**Comment by Jaykul** [comment]
yeah, I'm actually having trouble getting even the Windows version to work with the 6.0.0 package from MyGet

**Comment by Jaykul** [comment]
Ok, this is fixed in 1.0.0
